builtin/logical/pki/acme_challenges.go

```diff
@@ -47,12 +50,33 @@ func ValidateSHA256KeyAuthorization(keyAuthz string, token string, thumbprint st
 	return true, nil
 }
 
+func buildResolver(config *acmeConfigEntry) (*net.Resolver, error) {
```

Comment:
Are we planning on future errors being returned?

Reply:
:shrug: I dunno! Maybe in the future with multiple lists... This is just a weird API and weird set of failure modes, the caller finds out that this is a bad address, not us. :cry: